app/_src/using-mesh/managing-ingress-traffic/delegated.md

```diff
@@ -15,6 +15,10 @@ At a technical level the delegated gateway sidecar is similar to any other sidec
 in the mesh except that incoming traffic bypasses the sidecar and directly reaches the
 gateway.
 
+<center>
+<img src="/assets/images/docs/diagram-delegated-gateway-detailed@3x.jpg" alt="Delegated gateway" style="width: 387px; high: 452px; padding-top: 20px; padding-bottom: 10px;"/>
```

Comment:
not sure if strictly setting the width / height is good for responsive images but probably @johncowen or @kleinfreund would know better

Reply:
Just a note before I look further into this. There is a typo here so this probably doesn’t behave as you expect:

```suggestion
<img src="/assets/images/docs/diagram-delegated-gateway-detailed@3x.jpg" alt="Delegated gateway" style="width: 387px; height: 452px; padding-top: 20px; padding-bottom: 10px;"/>
```

Reply:
For comparison, how it looks without any of the style additions here:

![image](https://github.com/kumahq/kuma-website/assets/5774638/591f0cf2-29de-4bf5-a78b-28ca590364a3)

I think we’re better off not setting custom styles like that. There shouldn’t be a custom vertical padding here as other images don’t have that either. The `high` property, on account of it being a typo, doesn’t have an effect (and were it `height`, it would distort the image). That leaves us with `width` which has the effect of scaling down the image (to be smaller than in my screenshot here). We might as well scale down the actual image rather than doing it with CSS if we want it to always look smaller than what the diagram looks from the get go.